plugins/oauth/web_client/stylesheets/login.styl

```diff
@@ -29,15 +31,33 @@
     border-right 1px solid white
   .g-oauth-button-text
     display inline-block
-    padding 0 20px
+    text-align center
+    width 100px
```

Comment:
I say leave it alone for now. We can always tweak things once the larger names actually exist.